src/Mocks/owner_info.js

```diff
@@ -0,0 +1,19 @@
+export default {
```

Comment:
We'll calling Instagram API endpoints so we don't need this mock data.

Reply:
It's just for development purpose because hitting API every-time during development is not efficient. We will remove it.